plugins/comments/comments_2.0.md

```diff
@@ -86,7 +85,13 @@ The editor needs to be configured to highlight the commented text. The following
 ```js
 tinymce.init({
 ...
- content_style: '.mce-annotation { background: #fff0b7; } .tc-active-annotation {background: #ffe168; color: black; }',
+ content_style: '.tox-comments-visible span.tox-comment
```

Comment:
This needs to be on one line, or use string concatenation

Reply:
Fixed